index.js

```diff
@@ -15,20 +19,46 @@ const path = require("path");
 const db = require("./model/db");
 const verseInfo = require("./routes/verseInfo");
 const tafsir = require("./routes/tafsir");
+const testRoutes = require("./routes/testRoutes");
```

Comment:
~/testRoutes~ --> /auth

Reply:
I left the comment on the wrong endpoint. We don't need the /test endpoint at all, we need to move auth endpoints to a separate `/auth` endpoint.